cirq/google/__init__.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Classes for compiling, serializing, and using Google's Quantum Engine API."""
```

Comment:
This doesn't read well due to missing objects: what are we compiling and serializing? Suggestion:

```python
"""Classes for working with Google's Quantum Engine API."""
```

Reply:
Done.